tools/isobuild/import-scanner.js

```diff
@@ -900,18 +927,16 @@ export default class ImportScanner {
     // missing dependency so that we can include it in the app bundle.
     if (parentFile) {
       const missing =
-        parentFile.missingNodeModules ||
+        parentFile.missingModules ||
         Object.create(null);
       missing[id] = info;
-      parentFile.missingNodeModules = missing;
+      parentFile.missingModules = missing;
     }
 
-    if (! has(this.allMissingNodeModules, id) ||
-        ! info.possiblySpurious) {
-      // Allow any non-spurious identifier to replace an existing
-      // possibly spurious identifier.
```

Comment:
Since we're now tracking all failed imports separately, we don't have to be careful about letting non-spurious identifiers take precedence, since multiple `info` objects can coexist for the same identifier.